CONTRIBUTING.md

```diff
@@ -23,3 +23,20 @@ Try to respect the code style of the code you modify.
 For new Objective-C code or significant refactoring, it is advised to adopt the [Spotify Objective-C Coding Style](https://github.com/spotify/ios-style).
 
 For Swift code, please try to follow the [GitHub Swift Style Guide](https://github.com/github/swift-style-guide).
+
+## Commit messages
+
+1. Separate subject line from body with a blank line.
+
+2. Use present-tense, imperative-style for the subject line (as if you are giving orders to the codebase to change its behavior). This is shorter and consistent with commit texts automatically generated by Git.
+
+3. Try to limit the subject line to 50 characters : consistent commit histories are easier to read. Avoid at all costs subject lines with more than 72 characters : Github will truncate them with an ellipsis.
+
+4. Capitalize the subject line.
```

Comment:
The only request I have is that you change this to "Capitalise" as none of us are American 😛

Reply:
AE is Apple’s preferred variant though. I’m for consistency!

Reply:
And that's how the virus starts to spread...